<env.mdp.mdp_wrapper.MDPWrapper object at 0x7fce3b2afb00>
Traceback (most recent call last):
  File "run_nash_dqn_exploiter_simple_mdp.py", line 31, in <module>
    model1 = eval(args.algorithm)(env, args)
  File "/home/quantumiracle/research/MARS/mars_core/rl/algorithm/nash_dqn_exploiter.py", line 21, in __init__
    super().__init__(env, args)
  File "/home/quantumiracle/research/MARS/mars_core/rl/algorithm/dqn.py", line 19, in __init__
    self.model = self._select_type(env, args).to(self.device)
  File "/home/quantumiracle/anaconda3/envs/res/lib/python3.6/site-packages/torch/nn/modules/module.py", line 865, in to
    return self._apply(convert)
  File "/home/quantumiracle/anaconda3/envs/res/lib/python3.6/site-packages/torch/nn/modules/module.py", line 536, in _apply
    module._apply(fn)
  File "/home/quantumiracle/anaconda3/envs/res/lib/python3.6/site-packages/torch/nn/modules/module.py", line 536, in _apply
    module._apply(fn)
  File "/home/quantumiracle/anaconda3/envs/res/lib/python3.6/site-packages/torch/nn/modules/module.py", line 536, in _apply
    module._apply(fn)
  File "/home/quantumiracle/anaconda3/envs/res/lib/python3.6/site-packages/torch/nn/modules/module.py", line 559, in _apply
    param_applied = fn(param)
  File "/home/quantumiracle/anaconda3/envs/res/lib/python3.6/site-packages/torch/nn/modules/module.py", line 863, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
RuntimeError: CUDA error: out of memory
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
